templates/konvoy-ui.yaml

```diff
@@ -0,0 +1,39 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1alpha1
+kind: Addon
+metadata:
+  name: konvoy-ui
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: konvoy-ui
+  annotations:
+    appversion.kubeaddons.mesosphere.io/kommander: "1.40.0"
+    endpoint.kubeaddons.mesosphere.io/landing: /landing
```

Comment:
This looks different than the endpoint annotations in other templates - is the `/landing` postfix unique to konvoy-ui? Others seem to use the `kubeaddons.mesosphere.io/name` label value